.devcontainer/docker-compose.override.yaml

```diff
@@ -0,0 +1,43 @@
+name: lumigator
+
+services:
+
+  backend:
+    build:
+      context: .
+      dockerfile: "Dockerfile"
+    platform: linux/amd64
+    command: ["uv", "run", "uvicorn", "backend.main:app", "--host", "0.0.0.0", "--port", "8000",  "--reload"]
+    depends_on:
+      - localstack
+    ports:
+      - 80:8000
```

Comment:
What is the rationale behind mapping to port 80 for local development and keeping port 8000 otherwise?
If we want to keep them different (my 2 cents is we could use the same port btw), it might be helpful to make it explicit in our documentation

Reply:
It's actually a great point Davide. I think it's a good moment to change this! I didn't want to change 80 for our dev environment because I know some scripts we use for testing (like the ones you shared in slack) point directly to localhost, but it would be a great moment to choose a port that works for us and use it everywhere! Should I use 8000 everywhere? or something more random defined by us like 5151?

Reply:
Thanks! I think 8000 is fine, as long as it is well documented so ppl know where to point their code and where to change this value in case they have something else running on the same port

Reply:
Updated Readme and consolidated port 8000 for local & Dev